src/Surfnet/ServiceProviderDashboard/Domain/ValueObject/Issue.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/**
+ * Copyright 2019 SURFnet B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Surfnet\ServiceProviderDashboard\Domain\ValueObject;
+
+class Issue
```

Comment:
This VO should be JSON serializable, when saving this file to file in the issues.json file, it is saved as an empty object. Resulting in fatal errors on the user side.

Reply:
The actual issue data was not stored in the JSON file, only the key was. I added jsonSerialize to the Issue type, this did the trick.